dubbo-compiler/src/main/resources/ReactorDubbo3TripleStub.mustache

```diff
@@ -54,6 +54,7 @@ public final class {{className}} {
     private static final StubServiceDescriptor serviceDescriptor = new StubServiceDescriptor(SERVICE_NAME,{{interfaceClassName}}.class);
 
     static {
+        org.apache.dubbo.rpc.protocol.tri.service.SchemaDescriptorRegistry.addSchemaDescriptor(SERVICE_NAME,{{outerClassName}}.getDescriptor());
```

Comment:
Does this mean that `outClassName` must appear in the `.proto` definition file?

Reply:
@caoyanan666 What if user does not set `outClassName` in his `.proto` file.

Reply:
Yes, must definition.  https://github.com/apache/dubbo/blob/3.2/dubbo-compiler/src/main/resources/Dubbo3TripleStub.mustache also requires definition outClassName 
![image](https://github.com/apache/dubbo/assets/55247691/88ee7507-1fe5-4654-b7e3-d92e0a734b5f)